src/py/py_spline.cpp

```diff
@@ -106,7 +107,7 @@ void PySpline::NewCore(const py::kwargs& kwargs) {
       knot_vector.reserve(n_knots);
 
       int nknots{0};
-      prev_knot = -1.;
+      prev_knot = std::numeric_limits<double>::min();
```

Comment:
Actually, one should use lowest() here, right? Otherwise a knot vector that contains min() would not work...